Web/Admin.UI/src/app/components/tenant/facility-view/resource-pie-chart/resource-pie-chart.component.ts

```
@@ -0,0 +1,116 @@
+  @ViewChild('chart', { static: true }) chartElement!: ElementRef<SVGSVGElement>;
+  @Input() data: IResourceCountSummary[] = [];
+
+  private svg: any;
+  private color: any;
+  private arc: any;
+  private pie: any;
+  private width = 700;
+  private height = 400;
+  private radius = Math.min(this.width, this.height) / 2;
+
+  constructor() { }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)